packages/pebble-web/src/components/DateInput.tsx

```diff
@@ -107,7 +108,7 @@ export default class DateInput extends React.PureComponent<
                   ...(inputProps && inputProps.inputProps),
                   onChange
                 }}
-                className={inputStyle}
+                className={cx(inputStyle, inputProps && inputProps.className)}
```

Comment:
`DateInputProps` has no `className` field

https://github.com/anarock/pebble/blob/f1409fcd3a2a35d57988f0f519c4c0e6fe6c32ef/packages/pebble-web/src/components/typings/DateInput.ts#L5-L11

Reply:
so?

Reply:
- `inputProps` in expression `inputProps && inputProps.className` is of type `DateInputProps`
- `DateInputProps` has no `className` field
- so `inputProps.className` should give TS error

Reply:
no `inputProps` is of type `Omit<SimpleInputProps, "value" | "onChange" | "placeholder">`